adapters/rubicon/rubicon.go

```diff
@@ -52,9 +52,15 @@ type rubiconParams struct {
 	Visitor   json.RawMessage `json:"visitor"`
 }
 
+type rubiconImpExtRPTrack struct {
```

Comment:
Please also update the json-schema in `static/bidder-params/rubicon.json`, so that the `/bidders/params` endpoint stays in sync with the code.

I'm planning to enforce this for my work on #150

Reply:
None of the new params are going to be Rubicon bidder params. Please correct me if I'm missing something here but I don't see any necessary changes to the RP bidder json schema.

Reply:
nope! My mistake... you're totally right. These are outgoing params to rubicon's server... not incoming ones to PBS.